test.cpp

```diff
@@ -0,0 +1,6 @@
+#include <iostream>
+
+int main(int argc, char **argv){
```

Comment:
Going forward, let's try to add spaces between the ) and the {. I know I would have annoyed my co-workers if I did that.

I don't care for this temporary file, but something to keep in mind in the future.

Reply:
We should configure a linter for checking this.

Reply:
Agreed!